common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -222,6 +225,26 @@ public static void load(String name, ClassLoader loader) {
         }
     }
 
+    private static URL getLibraryUrl(ClassLoader loader, String path) {
```

Comment:
We usually not have get prefixes for this kind of stuff... so maybe just call it `libraryUrl` ?